Assets/Scripts/InputManager/Actions/FireAction.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Player;
+
+namespace Inputs
+{
+    public class FireAction : InputAction
+    {
+        public override void Execute()
+        {
+            PlayerManager.Instance.playerController.setFireState();
```

Comment:
this will only work for 1 player - not flexible to work with N players

Reply:
Will change it for multiple players